benchmarks/block_store.py

```diff
@@ -288,10 +288,10 @@ async def run_add_block_benchmark(version: int):
             print("profiling get_full_blocks_at")
 
         start = monotonic()
-        for h in range(1, block_height):
-            blocks = await block_store.get_full_blocks_at([h])
+        for hi in range(1, block_height):
+            blocks = await block_store.get_full_blocks_at([hi])
```

Comment:
This expects a `uint32` but `hi` would be an `int`.  Anyways, this is progress and further fixes can be taken separately.  I would note that i would rather see `height` and `header_hash` above or such than `h` and `hi`.

This is not complained about since `BlockStore.create()` has no return type hint and thus `block_store` is `Any`.

No request for change here as smaller PRs are useful and there's no real expectation that this chain will end.